tsconfig.json

```diff
@@ -1,12 +1,17 @@
 {
     "compilerOptions": {
         "baseUrl": ".",
-        "paths": { "*": ["types/*"] },
+        "paths": {
+            "*": [
+                "types/*"
+            ]
+        },
         "module": "commonjs",
         "target": "es6",
         "outDir": "out",
         "lib": [
```

Comment:
We can remove the `lib` section. If `lib` does not exist, and `target` is `es6`, the following libs will be injected:

* `DOM`
* `ES6`
* `DOM.Iterable`
* `ScriptHost`

See https://www.typescriptlang.org/docs/handbook/compiler-options.html